config/foreman.hiera/scenario/katello.yaml

```diff
@@ -1,2 +1,19 @@
 ---
 puppet::server::foreman::katello: true
+
+iop_container_tag: 'foreman-3.16'
+
+iop::core_kafka::image: 'quay.io/strimzi/kafka:latest-kafka-3.7.1'
```

Comment:
This is the same as we have in the Puppet module at https://github.com/theforeman/puppet-iop/blob/master/manifests/core_kafka.pp#L12

Not blocking, but this lives in a different namespace and follows different naming structure, which seems very odd to me.

Reply:
Which part seems odd?